From b68dcc00054875221da88dd9307939a434ea9d34 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 23 Oct 2012 15:53:19 +0800
Subject: [PATCH 082/602] ENGR00227841 build: update JB compile flag

Update here to align with HAL's change.
HAL(hardware/imx)commit num:
493fb7017ee5ef68cd0b481f8285e16826765148

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx5x/imx5x.mk | 6 +++---
 imx6/imx6.mk   | 5 ++++-
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index 70f5916..d6d9764 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -68,9 +68,9 @@ PRODUCT_PACKAGES += \
 
 # imx5x Hardware HAL libs.
 PRODUCT_PACKAGES += \
-	sensors.freescale			\
+	sensors.SMD				\
 	overlay.imx5x				\
-	lights.freescale			\
+	lights.imx5x				\
 	gralloc.imx5x				\
 	copybit.imx5x				\
 	hwcomposer.imx5x            		\
@@ -128,7 +128,7 @@ PRODUCT_PACKAGES += \
 
 # gps related lib
 PRODUCT_PACKAGES += \
-	gps.freescale 				\
+	gps.SMD 				\
 	Orion.ini 				\
 	AR1520A-Auto.img 			\
 	ingsvcd					\
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 45dd2e9..e7059e7 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -74,6 +74,7 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
 	sensors.SABRESD				\
 	sensors.ARM2				\
+	sensors.SABREAUTO			\
 	overlay.imx6				\
 	lights.imx6				\
 	gralloc.imx6				\
@@ -123,7 +124,9 @@ PRODUCT_PACKAGES += \
 
 # gps related lib
 PRODUCT_PACKAGES += \
-	gps.freescale				\
+	gps.SABRESD				\
+	gps.ARM2				\
+	gps.SABREAUTO				\
 	Orion.ini 				\
 	AR1520A-Auto.img 			\
 	ingsvcd 				\
-- 
1.8.0

